srcpkgs/btrfs-backup/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'btrfs-backup'
+pkgname=btrfs-backup
+_pkgname=btrfs_backup
```

Comment:
Instead of a custom variable, just could just use: `${pkgname/-/_}`

Reply:
Perfect. Didn't know this. Thanks!